cmd/release/release_info.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "1.2.1",
+    "version": "3.1.1",
```

Comment:
Em where is the version `3.1.1` coming from.. still see 1.2.1 on quay.io/redhat-certification/chart-verifier.

Reply:
Oops, hang over from testing.